### ISSUE TITLE
Plans to transition away from using @apply?

### ISSUE THREAD

**Issue by erikjung** [bigfishtv/turret#17]
Regarding this warning that PostCSS currently emits when processing the source files...

```
You are using @apply rule and custom property sets.
This feature won't be included in next the major release of postcss-cssnext. 
This most likely won't get any more support from browser vendors as the spec 
is yet considered deprecated and alternative solutions are being discussed. 
Read more about the reason here https://github.com/pascalduez/postcss-apply
```

Are there any plans to deprecate the `_mixins` in favor of something else? If future versions of cssnext are going to omit the `postcss-apply` plugin, then it seems that plugin would become a separate dependency in order to use the `--foo {}` and `@apply` syntax.

**Comment by scottdejonge** [comment]
Yep, this will be the focus of v5.0.0, to migrate to the latest PostCSS, postcss-next version dependancies.

Not sure the best route here. I was contemplating [PostCSS-mixins](https://github.com/postcss/postcss-mixins) for extended functionality. But this adds an opinionated dependancy which forces compilation for the CSS to be usable.

The other option is to remove the mixins altogether, in favour of the styles being applied via classes alone.

Open to ideas and feedback on what would be the best way forward here.

**Comment by vilbht** [comment]
@scottdejonge any ETA on v5?

**Comment by scottdejonge** [comment]
@vilav I've written some initial thoughts and ideas for v5.0.0 in #21 to help identify a roadmap of features, additions, and refactoring that needs to be completed in order to make turretcss simpler and more configurable.

**Comment by scottdejonge** [comment]
For those wanting to play around with v5.0.0 the first alpha is on the v5.0.0 branch, also available on npm with the version being v5.0.0-alpha1

**Comment by scottdejonge** [comment]
Resolved in v5.0.0
